instrumentation/jsp-2.3/jsp-2.3.gradle

```diff
@@ -15,7 +15,9 @@ dependencies {
   compileOnly group: 'javax.servlet.jsp', name: 'javax.servlet.jsp-api', version: '2.3.0'
   compileOnly group: 'javax.servlet', name: 'javax.servlet-api', version: '3.1.0'
 
-  testImplementation project(':instrumentation:servlet:servlet-3.0')
+  testImplementation project(':instrumentation:servlet:servlet-3:servlet-3.0')
+  testImplementation project(':instrumentation:servlet:servlet-3:servlet-3.1')
```

Comment:
Why both are needed here?

Reply:
Tomcat 7.X uses servlet 3.0, Tomcat 9.X uses 4.0 (I think). I need to include both instrumentation so that both `test` and `test -PtestLatestDeps=true` pass.